app/controllers/github_webhooks_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class GithubWebhooksController < ApplicationController
+  require 'faker'
```

Comment:
Do you need to require faker explicitly? It should be required automatically by bundler.

Reply:
You're right. This line should be removed.